packages/core/state/selection/reducer.ts

```diff
@@ -155,6 +161,10 @@ export default makeReducer<SelectionStateBranch>(
         }),
         [SET_SORT_COLUMN]: (state, action) => ({
```

Comment:
Similarly adding this here will help typescript yell at a future developer that might cause a typing issue

```suggestion
        [SET_SORT_COLUMN]: (state, action: SetSortColumnAction) => ({
```